fuse/types.go

```diff
@@ -35,6 +35,9 @@ const (
 	// EINVAL Invalid argument
 	EINVAL = Status(syscall.EINVAL)
 
+	// ENXIO No such device or address
+	ENXIO = Status(syscall.ENXIO)
```

Comment:
Re: https://github.com/hanwen/go-fuse/commit/3dbe3569afbd4fb9d8d06c76202bb77d9f3248b8#r127107015, should we refer to `syscall.ENXIO` directly? @hanwen

Reply:
Sure